RecoParticleFlow/PFClusterProducer/src/SpikeAndDoubleSpikeCleaner.cc

```diff
@@ -117,10 +117,55 @@ clean(const edm::Handle<reco::PFRecHitCollection>& input,
     }    
     const spike_cleaning& clean = _thresholds.find(hitlayer)->second;    
     if( rechit.energy() < clean._singleSpikeThresh ) continue;
+
+
+
+    //Fix needed for HF
+    float compsumE = 0.0;
+    if ((hitlayer==12 or hitlayer==11))
+	{
+	int predphi =2;
```

Comment:
Ok, in the next version there will be more verbose comments.  

This section checks if a rechit is from the HF region and if so it adds additional energy from the companion rechits (ie HF_Had <-> HF_EM) to the total surrounding energy